src/test/java/it/gov/pagopa/pu/bff/controller/OrganizationControllerTest.java

```diff
@@ -0,0 +1,65 @@
+package it.gov.pagopa.pu.bff.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.gov.pagopa.pu.bff.controller.generated.OrganizationsApi;
+import it.gov.pagopa.pu.bff.dto.generated.OrganizationDTO;
+import it.gov.pagopa.pu.bff.security.JwtAuthenticationFilter;
+import it.gov.pagopa.pu.bff.service.organization.OrganizationServiceImpl;
+import it.gov.pagopa.pu.bff.util.TestUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(value = OrganizationsApi.class, excludeFilters = @ComponentScan.Filter(
```

Comment:
because we are creating the API through openApiGenerate, we don't neet to create SpringTest, let's just build Mockito tests.
we will test the API through postman